backend/types/contentBlockType.ts

```diff
@@ -0,0 +1,18 @@
+type contentType = "text" |"image"|"video";
```

Comment:
```suggestion
export type ContentType = "text" | "image" | "video";
```

- does it make more sense to place this inline in line3?
- nitpick: spacing, capitalization

Reply:
Also when do we use interfaces vs types? Is types only when we need to take advantage of unions?
(cc @draymond63 if you have a style opinion)

Reply:
Tbh that's the way I've been using types. Default interface, types for union logic. I think that's generally the standard, I don't see too much of a reason to deviate